examples/sztp/README.md

```diff
@@ -300,7 +300,7 @@ lease {
   fixed-address 10.127.127.100;
   filename "grubx64.efi";
   option subnet-mask 255.255.255.0;
-  option sztp-redirect-urls "http://192.0.2.1/demo.sh";
+  option sztp-redirect-urls "http://bootstrap:9090/restconf/operations/ietf-sztp-bootstrap-server:get-bootstrapping-data";
```

Comment:
Yes, I like using hostnames rather than IP addresses here.

Reply:
docker gives us DNS resolution for free